Sources/TuistKit/Generator/GraphManifestLoader.swift

```diff
@@ -104,10 +107,13 @@ class GraphManifestLoader: GraphManifestLoading {
     /// - Parameters:
     ///   - resourceLocator: Resource locator to look up Tuist-related resources.
     ///   - helpersLoader: Instance to compile and return a temporary module that contains the helper files.
+    ///   - manifestFilesLocator: Utility to locate manifest files.
     init(resourceLocator: ResourceLocating = ResourceLocator(),
-         projectDescriptionHelpersBuilder: ProjectDescriptionHelpersBuilding = ProjectDescriptionHelpersBuilder()) {
+         projectDescriptionHelpersBuilder: ProjectDescriptionHelpersBuilding = ProjectDescriptionHelpersBuilder(),
+         manifestFilesLocator: ManifestFilesLocating = ManifestFilesLocator()) {
         self.resourceLocator = resourceLocator
         self.projectDescriptionHelpersBuilder = projectDescriptionHelpersBuilder
+        self.manifestFilesLocator = manifestFilesLocator
```

Comment:
The reason why I'm extracting the logic to locate the manifest into its own utility class is because I plan to make some changes in the [location of the manifests](https://github.com/tuist/tuist/issues/693) and having it extracted makes it possible to use the logic from anywhere.